dtls/src/crypto/mod.rs

```diff
@@ -15,101 +15,47 @@ use der_parser::{oid, oid::Oid};
 use rcgen::KeyPair;
 use ring::rand::SystemRandom;
 use ring::signature::{EcdsaKeyPair, Ed25519KeyPair, RsaKeyPair};
+use std::convert::TryFrom;
 use std::sync::Arc;
 
-#[derive(Clone, PartialEq)]
+/// A X.509 certificate(s) used to authenticate a DTLS connection.
+#[derive(Clone, PartialEq, Debug)]
 pub struct Certificate {
+    /// DER-encoded certificates.
     pub certificate: Vec<rustls::Certificate>,
```

Comment:
btw, I'm still confused as to why we have a list of certificates here. I've checked all the places and we only create a single certificate.